### PR TITLE
Improve tests output transform cs

### PR DESCRIPTION
**Pull request by emilioward**
Changed tests to use PyTest that displays a more verbose output
WIP on the test_views file `images_test` & `html_test` 

## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by emilioward** [comment]
[Improve tests output in sdx-transform-cs (2)](https://trello.com/c/qtbAUBYf/444-improve-tests-output-in-sdx-transform-cs-2)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ### Unreleased
  - Add all service configurations to config file
+ - improve test output and implemented Pytest
```
Comment (gemmai95) [inline CHANGELOG.md]: capital letter

**Comment by insacuri** [comment]
The reduced coverage isn't an issue as well, as it's no longer checking the tests themselves for coverage, this is a more accurate measurement of what the coverage of the suite is.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ### Unreleased
  - Add all service configurations to config file
+ - Improve test output and implemented Pytest
```
Comment (JamesGardiner) [inline CHANGELOG.md]: `Improve test output and begin using pytest as default test runner`

File: Makefile [hunk 3]
```diff
@@ -6,7 +6,7 @@ build:
 test:
 	pip3 install -r test_requirements.txt
 	flake8 --exclude ./lib/*
-	python3 -m unittest tests/*.py
+	pytest -v --cov transform
```
Comment (JamesGardiner) [inline Makefile]: Having the html output locally could be useful.

**Pull request merged**